src/init.cpp

```diff
@@ -914,13 +915,25 @@ bool AppInitParameterInteraction()
             for (const auto& cat : categories) {
                 uint32_t flag;
                 if (!GetLogCategory(&flag, &cat)) {
-                    InitWarning(strprintf(_("Unsupported logging category %s.\n"), cat));
```

Comment:
To not give the translators extra work, please try to use the same translation message as below. You could add a %s for the option.